service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/SendNegativeAcknowledgementTaskDispatcherTest.java

```diff
@@ -1,94 +0,0 @@
-package uk.nhs.adaptors.gp2gp.ehr;
-
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.ArgumentCaptor;
-import org.mockito.Captor;
-import org.mockito.Mock;
-import org.mockito.junit.jupiter.MockitoExtension;
-import uk.nhs.adaptors.gp2gp.common.service.RandomIdGeneratorService;
-import uk.nhs.adaptors.gp2gp.common.task.TaskDispatcher;
-import uk.nhs.adaptors.gp2gp.common.task.TaskType;
-import uk.nhs.adaptors.gp2gp.ehr.model.EhrExtractStatus;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-@ExtendWith(MockitoExtension.class)
-public class SendNegativeAcknowledgementTaskDispatcherTest {
```

Comment:
The dispatcher is gone, but it would be worth keeping this test and adjusting it to `SendAcknowledgementTaskDispatcher.java`

Reply:
I though I have test for both cases, yes it should be added for SendAcknowledgementTaskDispatcher for positive and negative